core/com.b2international.snowowl.core/src/com/b2international/snowowl/core/similarity/Similarity.java

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright 2022 B2i Healthcare Pte Ltd, http://b2i.sg
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.b2international.snowowl.core.similarity;
+
+import java.util.Objects;
+
+import com.b2international.collections.PrimitiveLists;
+import com.b2international.collections.floats.FloatList;
+import com.b2international.index.Doc;
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+/**
+ * @since 8.3
+ */
+@Doc(nested = true)
+@JsonIgnoreProperties(ignoreUnknown = true)
+public final class Similarity {
+	
+	private final String source_value;
```

Comment:
Please use `camelCased` fields and getters (and some extra `JsonProperty` annotations on the getter methods if Jackson doesn't figure out what you want to call these JSON properties from the constructor alone).